src/dispatchers/touchDispatcher.ts

```diff
@@ -38,16 +38,14 @@ module Plottable.Dispatchers {
 
       this._translator = Utils.ClientToSVGTranslator.getTranslator(svg);
 
-      this._startCallbacks = new Utils.CallbackSet<TouchCallback>();
-      this._moveCallbacks = new Utils.CallbackSet<TouchCallback>();
-      this._endCallbacks = new Utils.CallbackSet<TouchCallback>();
-      this._cancelCallbacks = new Utils.CallbackSet<TouchCallback>();
-      this._callbacks = [this._moveCallbacks, this._startCallbacks, this._endCallbacks, this._cancelCallbacks];
-
-      this._eventToCallback["touchstart"] = (e: TouchEvent) => this._measureAndDispatch(e, this._startCallbacks, "page");
-      this._eventToCallback["touchmove"] = (e: TouchEvent) => this._measureAndDispatch(e, this._moveCallbacks, "page");
-      this._eventToCallback["touchend"] = (e: TouchEvent) => this._measureAndDispatch(e, this._endCallbacks, "page");
-      this._eventToCallback["touchcancel"] = (e: TouchEvent) => this._measureAndDispatch(e, this._cancelCallbacks, "page");
+      this._eventToProcessingFunction[Touch._TOUCHSTART_EVENT_NAME] =
```

Comment:
I don't think `scope` parameter is necessary in  `_measureAndDispatch ` as for current implementation, it's doing additional check for nothing; but I am also okay with leaving it there to be consistent with the Mouse Dispatcher